java/custom/org/mozilla/Test.java

```diff
@@ -0,0 +1,6 @@
+package org.mozilla;
```

Comment:
Works for me! I've done the same in the past with the Sha256 class (which I added to tests/ and not bench/).